### PR TITLE
Core 1796 create get eth balance module

### PR DESCRIPTION
**Pull request by isentropy**
- new GetEthBalance canvas module
- moved Web3j getWeb3j() to EthereumModuleOptions class to standardize Web3j connector generation

### CODE REVIEW

**Comment by hpihkala** [comment]
@isentropy could you take a look at the conflicts?

**Comment by isentropy** [comment]
> @isentropy could you take a look at the conflicts?
yes, mostly formatting changes. I'll fix

**Comment by isentropy** [comment]
@hpihkala I merged the formatting conflicts. How does the code look? Can I merge to master?

File: src/java/com/unifina/signalpath/blockchain/SolidityCompileDeploy.java [hunk 1]
```diff
@@ -165,7 +164,7 @@ protected BigInteger getGasLimit() {
 	}
 
 	protected Web3j getWeb3j() {
-		return Web3j.build(new HttpService(ethereumOptions.getRpcUrl()));
+		return ethereumOptions.getWeb3j(EthereumModuleOptions.RpcConnectionMethod.HTTP);
```
Comment (jtakalai) [inline src/java/com/unifina/signalpath/blockchain/SolidityCompileDeploy.java]: is there a specific reason for this to be HTTP?

Comment (isentropy) [reply src/java/com/unifina/signalpath/blockchain/SolidityCompileDeploy.java]: just didn't want to make any breaking changes. I can test WS and switch if you want. I'm not sure how web3j WS handles reconnection.

**Pull request merged**